.githooks/pre-push

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env sh
+RED="\033[1;31m"
+BLUE="\033[34m"
+GREEN="\033[1;32m"
+NOCOLOR="\033[0m"
+
+exit_code=1
+echo "${BLUE}(pre-push) Building backend...${NOCOLOR}"
+npm run build:backend
```

Comment:
Think we could also add the `npm run clean` script to ensure there aren't any undeleted files when running the tests

https://github.com/MarkBind/markbind/blob/b10e424352cb3041e8216444aae2cfa2debf6181/.githooks/post-checkout#L12